morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -796,7 +794,6 @@ public void prepareStatementParameters(NamedParameterPreparedStatement statement
 
       default:
         super.prepareStatementParameters(statement, values, parameter);
-        return;

```

Comment:
Cannot drop this due to our general PMD rules.
Need a `return` (preferred) or `break` (but then all cases should do `break` for consistency).

Reply:
Restored.